src/main/java/com/uid2/shared/model/ClientSideKeypair.java

```diff
@@ -0,0 +1,106 @@
+package com.uid2.shared.model;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Objects;
+
+public class ClientSideKeypair {
+
+    public static final int KEYPAIR_KEY_PREFIX_LENGTH = 9;
+
+    private final String subscriptionId;
+    private final byte[] publicKeyBytes;
+    private final byte[] privateKeyBytes;
```

Comment:
I have refactored this so that the constructor will just take the public and private keys as strings which include the prefixes. The constructor then stores PrivateKey and PublicKey objects and Strings with the prefixes. The information is recombined by calling keypair.encodePublicKeyToString()/encodePrivateKeyToString(). This way we can avoid exposing the byte array as well as directly provide the needed class.